src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyBlocksAndCatchBlocks.java

```diff
@@ -36,6 +36,50 @@ public int doSideEffect() {
   }
 
   public void emptyMethod() {}
+
+  void foo() throws Exception {
+    int a = 90;
+
+    if (a == 1) {
+    } else {
+    }
+    // false negative
```

Comment:
Why is this `false negative`? It's a valid case.

Reply:
removed false negative comments from formatted file